S/SLEEF/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SLEEF"
+version = v"3.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/shibatch/sleef.git", "8df2bce4596ec6b86f357beebf2e428564f0a4ec"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/sleef
+mkdir build-native
+cd build-native
+cmake \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_HOST_TOOLCHAIN} \
+    -G Ninja \
+    -DBUILD_TESTS=OFF \
+    -DBUILD_DFT=TRUE \
+    -DBUILD_QUAD=TRUE \
+    ..
+ninja all
+
+cd $WORKSPACE/srcdir/sleef
+if [[ $target == *w64-mingw32* ]]; then
+    atomic_patch -p1 ../patches/lowercase-windows-include.patch
+fi
+mkdir build-cross
+cd build-cross
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -G Ninja \
+    -DNATIVE_BUILD_DIR=$WORKSPACE/srcdir/sleef/build-native \
+    -DSLEEF_SHOW_CONFIG=1 \
+    -DBUILD_TESTS=OFF \
+    -DBUILD_DFT=TRUE \
+    -DBUILD_QUAD=TRUE \
+    ..
+ninja all
+ninja install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+filter!(p -> !(arch(p) == "armv6l" || arch(p) == "armv7l"), platforms) # armv6l and armv7l builds fail trying to force an architecture with -march
```

Comment:
The solution here is to remove the `-march` flags

Reply:
It seems there is some stuff happening in [Configure.cmake#L278](https://github.com/shibatch/sleef/blob/8df2bce4596ec6b86f357beebf2e428564f0a4ec/Configure.cmake#L278) - the patch I've included in https://github.com/JuliaPackaging/Yggdrasil/pull/4569/commits/45ecf4210055740b42d1ab86ceed2ea0531269de **might** fix it...